include/Cluster.h

```diff
@@ -62,7 +62,7 @@ class Cluster
   /// calculates all cluster properties and updates members
   /// @param Signal
   /// @todo Figure out how to properly calculate the sigmas of fiber and timeStamp
-  inline void addSignal(const Signal &signal);
+  void addSignal(const Signal &signal);
```

Comment:
In case you wondered: For some reason back in the day this did not compile for me without the `inline`. Weird but good that it is gone.